test/upload-download.test.js

```diff
@@ -128,6 +131,7 @@ describe('storage service', function() {
       .set('Accept', 'application/json')
       .expect('Content-Type', /json/)
       .expect(200, function(err, res) {
+        if (err) done(err);
         done();
```

Comment:
The same as https://github.com/strongloop/loopback-component-storage/pull/179/files#r90672793